pages/_app.tsx

```diff
@@ -103,7 +103,11 @@ const MyApp = (props: Props) => {
                           <Head>
                             <meta
                               name="viewport"
-                              content="width=device-width, user-scalable=no, initial-scale=1"
+                              content="width=device-width, user-scalable=no, initial-scale=1, viewport-fit=cover"
+                            />
+                            <meta
+                              name="apple-mobile-web-app-status-bar-style"
+                              content="black-translucent"
```

Comment:
Could you please remove the translucency here?

<img width="432" height="177" alt="Image" src="https://github.com/user-attachments/assets/7d3879e1-887f-4f58-a919-4c5f31434b60" />

For map it is useful to see the exact canvas below, like in other mobile map apps.